packages/ses/README.md

```diff
@@ -1,17 +1,30 @@
-# Secure EcmaScript (SES)
+# Secure ECMAScript (SES)
 
-Secure EcmaScript (SES) is a frozen environment for running EcmaScript
+Secure ECMAScript (SES) is an execution environment that provides fine-grained
+sandboxing with Compartments.
+
+* **Compartments** Compartments are separate execution contexts: each one has its own global object and global lexical scope.
+* **Frozen realm** Compartments share their intrinsics to avoid identity discontinuity. By freezing the intrinsics, SES removes programs abilities to interfere with each other.
+* **Strict mode** SES enables JavaScript strict mode that enhances security, for example by changing some silent errors into throw errors.
+* **POLA** (Principle of Least Authority) By default, Compartments receive no ambient authorithy. They are created without host-provided APIs, (for example no XMLHttpRequest).
```

Comment:
I'm not sure how to rephrase, but the point is that, for example, `XMLHttpRequest` is not provided implicitly. It may still appear in the start compartment, and it may be explicitly endowed to other compartments.

Reply:
Proposed https://github.com/Agoric/SES-shim/commit/0f59f87bdc0a965b9c2b4ab175db5b0652b273fd